ui/cypress/e2e/collectors.test.ts

```diff
@@ -253,4 +253,121 @@ describe('Collectors', () => {
       })
     })
   })
+
+  describe('configuring plugins', () => {
+    // fix for https://github.com/influxdata/influxdb/issues/15500
+    describe('configuring nginx', () => {
+      beforeEach(() => {
+        // These clicks launch move through configuration modals rather than navigate to new pages
+        cy.contains('Create Configuration')
+          .click()
+          .then(() => {
+            cy.contains('NGINX')
+              .click()
+              .then(() => {
+                cy.contains('Continue')
+                  .click()
+                  .then(() => {
+                    cy.contains('nginx').click()
+                  })
+              })
+          })
+      })
+
+      it('can add and delete urls', () => {
+        cy.getByTestID('input-field').type('http://localhost:9999')
+        cy.contains('Add').click()
```

Comment:
How would you feel about checking to see if the input-field was cleared after the click resolved?